api/src/main/kotlin/org/jetbrains/kotlinx/dl/api/core/layer/VariableDto.kt

```diff
@@ -0,0 +1,38 @@
+package org.jetbrains.kotlinx.dl.api.core.layer
+
+import org.jetbrains.kotlinx.dl.api.core.initializer.Initializer
+import org.jetbrains.kotlinx.dl.api.core.regularizer.Regularizer
+import org.jetbrains.kotlinx.dl.api.core.util.getDType
+import org.tensorflow.Operand
+import org.tensorflow.Shape
+import org.tensorflow.op.Ops
+import org.tensorflow.op.core.Variable
+
+public data class VariableDto(
```

Comment:
Added class to encapsulate all related to TensorFlow's Variable<Float> info, which is exposed by `ParametrizedLayer::variables` and no need to pass variable initializers and regularizers through `KGraph`. The main idea here is that layers already represent computation graph and all related info can be stored here

Reply:
Because of it, it is possible to remove `KGraph` argument from `Layer::build` method